CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Add ingest provider to search
 - Add image metadata validation on publish
 - Add url validation to editor anchor button
+- Added `Send and Publish` so that item can be first send and publishd from different desk.
```

Comment:
There are some spelling mistakes here. Is _"...first sent and published from different desks._" the correct sentence?